jetstream/errors.py

```diff
@@ -38,3 +38,30 @@ def __init__(self, normandy_slug, message="Experiment has high population."):
 class ExplicitSkipException(ValidationException):
     def __init__(self, normandy_slug, message="Experiment is configured with skip=true."):
         super().__init__(f"{normandy_slug} -> {message}")
+
+
+class ConfigurationException(Exception):
```

Comment:
maybe
```suggestion
class InvalidConfigurationException(Exception):
```
to make it more explicit?